GPU/Common/ShaderId.cpp

```diff
@@ -83,14 +82,12 @@ void ComputeVertexShaderID(VShaderID *id_out, u32 vertType, bool useHWTransform,
 		_assert_(hasNormal);
 	}
 
-	bool enableFog = gstate.isFogEnabled() && !isModeThrough && !gstate.isModeClear();
```

Comment:
Just a note: if we remove the flag from the frag shader too, we just have to make sure it isn't enabled in clearMode (should be easy, already has a bit.)  Note that clear can be used in transform mode.

-[Unknown]

Reply:
Yup, so I noticed when I tried earlier today, and decided to not bother for now :)